src/bootstrap.js

```diff
@@ -3,7 +3,6 @@ import ReactDOM from 'react-dom';
 import AppEntry from './AppEntry';
 
 const root = document.getElementById('root');
+root.setAttribute('data-ouia-safe', true);
 
-ReactDOM.render(<AppEntry />, root, () =>
-  root.setAttribute('data-ouia-safe', true)
-);
+ReactDOM.createRoot(root).render(<AppEntry />);
```

Comment:
This file is useless. It has to exist but this code is ever used. It can be blank.